### PR TITLE
hugo: update to 0.102.2.

### PR DESCRIPTION
**Pull request by dataCobra**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64 (crossbuild)
  - i686 (crossbuild)

### CODE REVIEW

**Comment by classabbyamp** [comment]
this release changes nothing for us: https://github.com/gohugoio/hugo/compare/v0.102.0...v0.102.1

**Comment by dataCobra** [comment]
I've changed to version `0.102.0`. Because the new minor version is only related to deb-packages.

I've also added shell completions and man-pages.

**Comment by dataCobra** [comment]
@thypon, are you okay with the update and changes?

File: srcpkgs/hugo/template [hunk 1]
```diff
@@ -10,8 +10,18 @@ maintainer="Andrea Brancaleoni <abc@pompel.me>"
...
 post_install() {
 	vdoc README.md
+
+	${DESTDIR}/usr/bin/hugo gen man
+	for page in $(ls man/); do
+		vman man/$page
+	done
+
+	for shell in bash fish zsh; do
+		${DESTDIR}/usr/bin/hugo completion $shell > ${pkgname}.$shell
```
Comment (classabbyamp) [inline srcpkgs/hugo/template:24]: same thing

File: srcpkgs/hugo/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'hugo'
 pkgname=hugo
-version=0.101.0
+version=0.102.0
```
Comment (classabbyamp) [inline srcpkgs/hugo/template:3]: ah sorry i thought this was bumping 102 to 102.1, it's fine to go to .1 i guess

Comment (dataCobra) [reply srcpkgs/hugo/template:3]: No worries. :) I'll change it back to 0.102.1.

File: srcpkgs/hugo/template [hunk 3]
```diff
@@ -10,8 +10,18 @@ maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="Apache-2.0"
 homepage="https://gohugo.io"
 distfiles="https://github.com/gohugoio/hugo/archive/v${version}.tar.gz"
-checksum=ce5e2c37d9980428cfbfb22cabedc29aebe8f1142ce261777d0435f9f2d6d1cb
+checksum=b2f231b1abc8e260cc241a1622da2a84083f27041b6d14a7e769180740512608
 
 post_install() {
 	vdoc README.md
+
+	${DESTDIR}/usr/bin/hugo gen man
```
[the remark this answers is not in the document]
Comment (dataCobra) [reply srcpkgs/hugo/template:18]: Ah that's the reason why it's failing. Thanks for the info. :+1:

[the remark this answers is not in the document]
Comment (dataCobra) [reply srcpkgs/hugo/template:18]: Hey @classabbyamp,

what do I need to do additionally?

I added `build_helper=qemu` but aarch64 does still fail with the same error.

[the remark this answers is not in the document]
Comment (dataCobra) [reply srcpkgs/hugo/template:18]: Maybe I found the solution.

**Comment by dataCobra** [comment]
Everything is fine know. :slightly_smiling_face: 

@classabbyamp I hope this template setup is fine?

**Pull request merged**